languages/javascript/concepts/equality/introduction.md

```diff
@@ -0,0 +1,66 @@
+**Equality** in the javascript can be referred in two ways which is `==` or `===`
```

Comment:
Suggestion is to change this sentence. Recommendation:

**Equality** in JavaScript can be referred in two ways:  `==` or `===`